src/lib/components/data-encoder-settings.svelte

```diff
@@ -26,11 +26,14 @@
 </script>
 
 <script lang="ts">
+  import RadioGroup from '$lib/holocene/radio-input/radio-group.svelte';
+  import RadioInput from '$lib/holocene/radio-input/radio-input.svelte';
+
   let endpoint = $codecEndpoint ?? '';
   let port = $dataConverterPort ?? '';
   let passToken = $passAccessToken ?? false;
   let includeCreds = $includeCredentials ?? false;
-  let override = $overrideRemoteCodecConfiguration ?? false;
+  let override = writable($overrideRemoteCodecConfiguration);
```

Comment:
Curious why you've chosen `writable` over ⬇️ here
```suggestion
  let override = $overrideRemoteCodecConfiguration ?? false;
```
and if we want to change the other variables to stores as well 🤔

Reply:
I did it this way because `override` needs to be a `Writable`, and `let override = $overrideRemoteCodecConfiguration` would make `override` the value of the store, i.e. a `boolean`. Additionally, `let override = overrideRemoteCodecConfiguration` (without the `$`) assign `override` by reference and thus the store value would still be mutated.

Reply:
Ah forgot about the context r.e. the group needing to be a store in the note for this PR, thanks!